.gitignore

```diff
@@ -0,0 +1,34 @@
+*/.gitignore
```

Comment:
would you be able to add spaces between the sections? so it's easier to read and also consistent formatting with other .gitignore files in our android repo
https://github.com/android/wear-os-samples/blob/master/.gitignore

https://github.com/google-developer-training/basic-android-kotlin-compose-training-amphibians/pull/4/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947